backend/app/utils/jsonl_usage_examples.py

```
@@ -0,0 +1,367 @@
+
+本文件展示了如何使用新的 JSONL 解析器系统，包括：
+- 基础使用方法
+- 高级配置选项
+- 自定义扩展
+- 错误处理
+- 性能优化
+"""
+
+import asyncio
+import logging
+from typing import List, Any, Dict
```

Ruff (0.11.9)

`typing.List` is deprecated, use `list` instead

(UP035)